libantworld/route_ardin.cc

```diff
@@ -204,8 +189,8 @@ bool RouteArdin::load(const std::string &filename, bool realign)
             const radian_t heading = m_Headings[i - 1];
 
             // Realign segment to this angle
-            waypoint[0] = prevWaypoint[0] + (0.1f * cos(heading));
-            waypoint[1] = prevWaypoint[1] - (0.1f * sin(heading));
+            waypoint[0] = prevWaypoint[0] + (0.1f * units::math::cos(heading));
+            waypoint[1] = prevWaypoint[1] - (0.1f * units::math::sin(heading));
```

Comment:
I suppose it should probably be ``0.1`` rather than ``0.1f`` here. I'm sure the units library would complain if it didn't like casting a float here, but it is using doubles internally.